Makefile

```diff
@@ -48,10 +53,10 @@ clean-cronjobs: options.env
 	oc process -f templates/cronjobs.yaml --param-file options.env  | oc delete -f -
 
 roles: options.env
-	oc process -f templates/roles.yaml -p NAMESPACE=`oc project -q` | oc apply -f -
+	oc process -f templates/roles.yaml --param-file options.env | oc apply -f -
 
 clean-roles: options.env
-	oc process -f templates/roles.yaml -p NAMESPACE=`oc project -q` | oc delete -f -
+	oc process -f templates/roles.yaml --param-file options.env | oc delete -f -
```

Comment:
This won't work actually unless you add all the other variables to this template, even though they are not used. `oc` won't allow undeclared params in the file.

Reply:
By just having the namespace in the options.env I was able to run
`make cronjobs`
`make running`
`make roles`

If I add more to the options.env though, then make running and make roles will fail...
The flip side is twice I ran `make running` and `make hibernating` in the wrong namespace... And they will fail, because the serviceAccount is not present there.  So its a tradeoff.

Reply:
Actually I found a solution, using:  --ignore-unknown-parameters=true, this will ignore the parameters that don't apply.

Reply:
Cool find with `--ignore-unknown-parameters=true`! Do you want to remove the unused params that are declared in the `hibernating-job.yaml` and `running-job.yaml` templates then?